build.gradle.kts

```diff
@@ -0,0 +1,372 @@
+        implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion")
+        testImplementation("com.nhaarman.mockitokotlin2:mockito-kotlin:$mockitoKotlinVersion")
+        testImplementation("org.mockito:mockito-core:$mockitoVersion")
+        testImplementation("org.assertj:assertj-core:$assertjVersion")
+        testImplementation("junit:junit:$junitVersion")
+    }
+
+    plugins.withType<TestLoggerPlugin> {
+        configure<TestLoggerExtension> {
+            showFullStackTraces = true
+            showStandardStreams = true
+            showPassedStandardStreams = false
+            showSkippedStandardStreams = true
+            showFailedStandardStreams = true
+        }
+    }
+
+    tasks.test {
+        configure<JacocoTaskExtension> {
+            // don't instrument sdk, icons, ktlint, etc.
+            includes = listOf("software.aws.toolkits.*")
+            excludes = listOf("software.aws.toolkits.ktlint.*")
+        }
+
+        reports {
+            junitXml.isEnabled = true
+            html.isEnabled = true
+        }
+    }
+
+    plugins.withType<IdeaPlugin> {
+        model.module.apply {
+            sourceDirs.plusAssign(sourceSets.main.get().java.srcDirs)
+            resourceDirs.plusAssign(sourceSets.main.get().resources.srcDirs)
+            testSourceDirs.plusAssign(File("tst-$ideVersion"))
+            testResourceDirs.plusAssign(File("tst-resources-$ideVersion"))
+
+            sourceDirs.minusAssign(File("it"))
+            testSourceDirs.plusAssign(File("it"))
+            testSourceDirs.plusAssign(File("it-$ideVersion"))
+
+            resourceDirs.minusAssign(File("it-resources"))
+            testResourceDirs.plusAssign(File("it-resources"))
+            testResourceDirs.plusAssign(File("it-resources-$ideVersion"))
+        }
+    }
+
+    tasks.register<Test>("integrationTest") {
+        group = LifecycleBasePlugin.VERIFICATION_GROUP
+        description = "Runs the integration tests."
+        testClassesDirs = sourceSets["integrationTest"].output.classesDirs
+        classpath = sourceSets["integrationTest"].runtimeClasspath
+
+        configure<JacocoTaskExtension> {
+            // don"t instrument sdk, icons, ktlint, etc.
+            includes = listOf("software.aws.toolkits.*")
+            excludes = listOf("software.aws.toolkits.ktlint.*")
+        }
+
+        project.plugins.withId("org.jetbrains.intellij") {
+            systemProperty("log.dir", "${(project.extensions["intellij"] as org.jetbrains.intellij.IntelliJPluginExtension).sandboxDirectory}-test/logs")
+        }
+
+        mustRunAfter(tasks.test)
+    }
+
+    project.plugins.withId("org.jetbrains.intellij") {
+        tasks.withType(org.jetbrains.intellij.tasks.DownloadRobotServerPluginTask::class.java) {
+            this.version = remoteRobotVersion
+        }
+
+        tasks.withType(org.jetbrains.intellij.tasks.RunIdeForUiTestTask::class.java).all {
```

Comment:
replace with `tasks.withType<RunIdeForUiTestTask>()` - it's already been imported so you shouldn't need the fully qualified class name.

Reply:
The auto import is wonky, fixed